app/_data/docs_nav_kgo_1.0.x.yml

```
@@ -26,8 +26,6 @@ items:
         absolute_url: true
       - text: Support
```

```suggestion
      - text: Version Support Policy
```
This might work, to keep in line with what we have for Kong Gateway. And "Support" could mean something like about the Kong Support team.

---

Done that in 504d943b1ecae60936e8a24071f3aca38c928111 to also change it in other versions' TOCs.